src/libcore/macros.rs

```diff
@@ -26,31 +26,29 @@ macro_rules! panic {
 /// For details, see `std::macros`.
 #[cfg(not(bootstrap))]
 #[macro_export]
-#[allow_internal_unstable(core_panic, panic_internals)]
+#[allow_internal_unstable(core_panic,
+    // FIXME(anp, eddyb) `core_intrinsics` is used here to allow calling
+    // the `caller_location` intrinsic, but once  `#[track_caller]` is implemented,
+    // `panicking::{panic, panic_fmt}` can use that instead of a `Location` argument.
+    core_intrinsics,
+)]
 #[stable(feature = "core", since = "1.6.0")]
 macro_rules! panic {
```

Comment:
btw, I think a previous PR forgot to change the macro in `libstd` (which is not the same as the one in libcore). cc @anp

Reply:
@anp told me it was intentional, as they use different entry-points, and we can probably wait to have full `#[track_caller]` before changing `libstd`.